docker/code_style/Dockerfile

```diff
@@ -5,4 +5,4 @@ RUN apk add --no-cache \
         perl \
         uncrustify \
```

Comment:
I think you want to install git here as well.

Reply:
Fixed 06f05651a4cd4b5f0b09fd908dfd23e4ac452a96